### ISSUE TITLE
Update SDK_RUNTIME_VERSION

### ISSUE THREAD

**Issue by helloworld-hash** [flathub/org.freedesktop.Platform.GL.nvidia#26]
build.sh have
SDK_RUNTIME_VERSION=1.6 on line 15
but org.freedesktop.Platform//1.6 is not maintained any more
should be updated to 18.08 or 19.08

**Comment by TingPing** [comment]
For practical purposes this means nothing. The driver isn't built against any runtime, doesn't pull any runtime into users, and 1.6 won't be deleted any time soon.

But there is no harm in getting updated eventually.

**Comment by helloworld-hash** [comment]
On flatpak version 1.4.3 
Installing driver pulls runtime too.

**Comment by helloworld-hash** [comment]
Here's how I installed it. 

```
[localhost ~]$ flatpak install org.freedesktop.Platform.GL.nvidia-430-40
Looking for matches…
Found similar ref(s) for ‘org.freedesktop.Platform.GL.nvidia-430-40’ in remote ‘flathub’ (system).
Use this remote? [Y/n]: y
Required runtime for org.freedesktop.Platform.GL.nvidia-430-40/x86_64/1.4 (runtime/org.freedesktop.Platform/x86_64/1.6) found in remote flathub
Do you want to install it? [Y/n]: y


        ID                                                   Arch              Branch           Remote            Download
 1. [✓] org.freedesktop.Platform                             x86_64            1.6              flathub           183.0 MB / 395.4 MB
 2. [✓] org.freedesktop.Platform.VAAPI.Intel                 x86_64            1.6              flathub             2.7 MB / 2.7 MB
 3. [✓] org.freedesktop.Platform.ffmpeg                      x86_64            1.6              flathub             2.9 MB / 2.9 MB
 4. [✓] org.freedesktop.Platform.Locale                      x86_64            1.6              flathub            14.3 kB / 177.1 MB
 5. [✓] org.freedesktop.Platform.GL.nvidia-430-40            x86_64            1.4              flathub            71.0 MB / 71.0 MB

Installation complete.

```

**Comment by TingPing** [comment]
Hmm, generally you never install it directly. No matter what that behavior will be wrong.

Maybe `flatpak` can change this behavior somehow. This extension is a very odd exception in how it works.

**Comment by helloworld-hash** [comment]
Yes I didn't install it directly, after installing nvidia-driver on host, when I updated flatpaks it only installed this extension without runtime. After updating again(2nd update) this time flatpak pulled runtime.

**Comment by TingPing** [comment]
Interesting, OK well still an issue to be reported against `flatpak/flatpak`

**Comment by TingPing** [comment]
I opened an issue over this: https://github.com/flatpak/flatpak/issues/3173

**Comment by SicLuceatLux** [comment]
Same here. Somehow I can not use steam anymore - com.steamsoftware.Steam wants 19.08 and does not install the nvidiadriver. Installing the driver manually pulls the runtime 1.6...cleaning with flatpak remove --unused removes them and update adds them again. Btw the same with Lutris. So Im supporting @helloworld-hash

**Comment by TingPing** [comment]
I get that it sucks but its a Flatpak bug, to be fixed in Flatpak.

**Comment by Maltimore** [comment]
It seems to have been fixed over at flatpak/flatpak#3173 so maybe this issue can be closed here?

**Comment by TingPing** [comment]
Yes and it isn't a problem in flatpak > 1.5.1
